src/alfasim_score/common.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+from typing import Dict
+
+from barril.units import Array
+from barril.units import Scalar
+
+
+def prepare_for_regression(values: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
def prepare_for_regression(values: dict[str, Any]) -> dict[str, Any]:
```

Reply:
I'm using this way to have support for Python 3.8, otherwise it breaks the build for this version.